testsuite/cluster-test/src/suite.rs

```diff
@@ -63,10 +64,10 @@ impl ExperimentSuite {
             PerformanceBenchmarkParams::new_nodes_down(10).build(cluster),
         ));
         experiments.push(Box::new(
-            PerformanceBenchmarkParams::new_fixed_tps(0, 10).build(cluster),
+            PerformanceBenchmarkThreeRegionSimulationParams {}.build(cluster),
```

Comment:
Is this change needed?

Reply:
just wanna make pre-release and perf suites have same order, I forgot to change perf-suite last time